[Freshness] PSScriptAnalyzer Article Updates - Batch 1 (#401)
MicrosoftDocs/PowerShell-Docs-Modules: reference/docs-conceptual/PSScriptAnalyzer/Rules/AvoidGlobalAliases.md

## Description

Global aliases can override existing aliases in the current session. This override

> creates naming conflicts that lead to hard to diagnose problems for module and script consumers.

Reviewer: Recommend updating this to use the same construction as `AvoidGlobalFUnctions` which reads more clearly to me:

```suggestion
Global aliases can unintentionally override existing aliases in the session, leading to unexpected
behavior and name collisions. Name collisions make it difficult for module consumers to diagnose
issues and maintain code reliability.
```

This rule is not available in PowerShell version 3 or 4 because it uses the
`StaticParameterBinder.BindCommand` API.

Instead of using the Global scope, use other scope modifiers such as `Local` or `Process` when

> creating new aliases. To learn more, see [about_Scopes][01].

Reviewer: ```suggestion
To avoid this issue, define aliases without the **Scope** parameter, or use other appropriate scope
modifiers. To learn more, see [about_Scopes][01].
```

## Description

Global aliases can unintentionally override existing aliases in the session, leading to unexpected
behavior and name collisions. Name collisions make it difficult for module consumers to diagnose

> issues and maintain code reliability.

Reviewer: I rather like the "This rule detects <thing>. <Explanation of why thing is bad>" construction used in the prior rule.

Suggest following that format for remaining rules.
```suggestion
This rule detects uses of the `New-Alias` command to create aliases in the global scope. Global
aliases can unintentionally override existing aliases in the session, leading to unexpected
behavior and name collisions. Name collisions make it difficult for module consumers to diagnose
issues and maintain code reliability.
```
